### ISSUE TITLE
All API endpoints down

### ISSUE THREAD

**Issue by richardtallent** [mxabierto/api#8]
Hello,

All API endpoints are currently responding with "502 Bad Gateway". For example, this URI:

https://api.datos.gob.mx/v1/api-catalog

I'm looking for the data behind the SINEC `Catálogo Mexicano de Normas`. There is a search interface here (https://www.sinec.gob.mx/SINEC/Vista/Normalizacion/BusquedaNormas.xhtml), but no means of downloading a full list of the standards, so I'm hoping it is available through the API.

**Comment by dicarlomagnus** [comment]
That's correct all these API endpoints are down right now: https://datos.gob.mx/blog/catalogo-apidatosgobmx?category=api-cdn&tag=nula

**Comment by dicarlomagnus** [comment]
> Hello,
> 
> All API endpoints are currently responding with "502 Bad Gateway". For example, this URI:
> 
> https://api.datos.gob.mx/v1/api-catalog
> 
> I'm looking for the data behind the SINEC `Catálogo Mexicano de Normas`. There is a search interface here (https://www.sinec.gob.mx/SINEC/Vista/Normalizacion/BusquedaNormas.xhtml), but no means of downloading a full list of the standards, so I'm hoping it is available through the API.

this was reported on Aug 31st and still is open...

**Comment by dicarlomagnus** [comment]
It's working now, seems like this service is not reliable. This is an important point to have in mind before using this API as a principal source of truth.

**Comment by richardtallent** [comment]
Thanks @dicarlomagnus! By "not reliable," are you referring to the uptime of the API, or the veracity of the dataset?

I was able to pull the list from the SINEC site directly, but it was a process I hope not to repeat in the future.
